index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="style.css" />
+		<title>Document</title>
+	</head>
+	<body>
+		<header>
+			<nav class="nav__menu">
+				<figure class="nav__logo">
+					<img src="./images/logo.svg" alt="logo" />
+				</figure>
+				<ul class="nav__list">
```

Comment:
Skoro jest Element z BEM to powinien być też Block. Obecnie w kodzie nie widzę klasy `nav`.
Więcej o BEM tutaj doczytasz: https://devmentor.pl/b/metodologia-bem-w-css-i-sass